name_that_hash/hashes.py

```diff
@@ -1641,4 +1641,59 @@ class HashInfo:
                 )
         ],
     ),
+    Prototype(
+    regex=re.compile(r"\$bitcoin\$[0-9]{2,4}\$[a-fA-F0-9$]{250,350}", re.IGNORECASE),
+    modes=[
+        HashInfo(
+            name="Bitcoin / Litecoin",
+            hashcat=11300,
+            john="bitcoin",
+            extended=False,
+            description="Use Bitcoin2John.py to extract the hash for cracking."
+            )
+        ],
+    ),
+    Prototype(
+    regex=re.compile(r"\$ethereum\$[a-z0-9*]{150,250}", re.IGNORECASE),
+    modes=[
+        HashInfo(
+            name="Ethereum PBKDF2",
```

Comment:
I think we should use the full names that HashCat uses since we did that with the Kerberos hash types too. So change it to `Ethereum Wallet, PBKDF2-HMAC-SHA256` instead of `Ethereum PBKDF2`.